### ISSUE TITLE
doVolcanoReport often fails to report correct civ

### ISSUE THREAD

**Issue by dzfischer** [caveman2cosmos/Caveman2Cosmos#1387]
`<Tag>TXT_KEY_EVENTTRIGGER_VOLCANO_ACTIVE</Tag>` has `<English>Reports from the %s1_civ_adjective empire speak of increased volcanic activity.</English>`

This often appears ingame as "Reports from the ??? empire..."
Similar occasionally occurs with other volcano reports civ-labeled reports.

I _think_ this occurs when a volcano event occurs in a tile that the player has previously seen (doVolcanoReport checks if tile is visible) but does not currently know who owns it due to fog of war.

Code section in CvRandomEventInterface.py:
```
doVolcanoReport((pPlot, BugUtil.getPlainText("TXT_KEY_EVENTTRIGGER_VOLCANO_ACTIVE")))
```

points to
```
def doVolcanoReport(argsList):
  pPlot = argsList[0]
  szBuffer = argsList[1]
  ft_volcano_dormant = GC.getInfoTypeForString('FEATURE_VOLCANO_DORMANT')
  ft_volcano_active = GC.getInfoTypeForString('FEATURE_VOLCANO_ACTIVE')

  # report message to any one who can see this plot
  iMaxPlayer = GC.getMAX_PC_PLAYERS()
  for i in xrange(iMaxPlayer):
    loopPlayer = GC.getPlayer(i)
    if loopPlayer.isHuman() and loopPlayer.isAlive() and pPlot.isVisible(loopPlayer.getTeam(), False):
      CyInterface().addMessage(loopPlayer.getID(), False, GC.getEVENT_MESSAGE_TIME(), szBuffer, "AS2D_BOMBARDED", InterfaceMessageTypes.MESSAGE_TYPE_INFO, GC.getFeatureInfo(ft_volcano_active).getButton(), GC.getCOLOR_RED(), pPlot.getX(), pPlot.getY(), True, True)
```

**Comment by Toffer90** [comment]
Whoever rewrote the volcano event last didn't set the text up to accept variables, I remember about 2-3 years ago someone jumped into C2C modding and did pretty much one thing, which was to rewrite the volcano event code in python.

I'm pretty unhappy with the volcano stuff in general so I'm likely to just delete it all if I were to engage with its implementation; so for now I'm keeping my distance.

**Comment by Toffer90** [comment]
I guess I could fix this little annoyance pretty quick though.

**Comment by Toffer90** [comment]
Fixed in 5f31153
